kymatio/scattering2d/tests/test_tensorflow_scattering2d.py

```diff
@@ -0,0 +1,366 @@
+import os
+import io
+import tensorflow as tf
+from kymatio.scattering2d import Scattering2D
+import torch
+import numpy as np
+import pytest
+from collections import namedtuple
+
+backends = []
+
+from kymatio.scattering2d.backend.tensorflow_backend import backend
+backends.append(backend)
+
+class TestPad:
```

Comment:
Alot of the tests in this file are exactly the same as the ones on the Numpy end. The only changes are tensorflow is written instead of numpy, and there's one small difference I'll point out in a comment.

Reply:
Fine

Reply:
Why was this done exactly? Why should we expect feeding NumPy arrays into the TF frontend to work?

Reply:
there are two modes with tf, but I'm not much familiar with it

Reply:
> I'm not much familiar with it

Is anyone around here? I'm serious. Do we know what we're doing?

Reply:
I find it a bit weird to try numpy tensors here but I'd like to have a fresh opinion from a tf user. I don't know any (👀)

Reply:
Ok. Will see if I can trick someone into looking at it. I think it's fine to merge, but before we go to `master`, someone should have looked at it and made sure it's ok.

Reply:
I'm familiar with TF at a basic level, it was used for the labs/programming assignments for my universities machine learning classes, however I've used Pytorch so much more in general.

I'd like to think that if we got this down on the tf front/backend (we're getting the same numbers as the torch end) we did some stuff right!